sirepo/job_api.py

```diff
@@ -111,6 +112,17 @@ def api_runCancel():
 @api_perm.require_user
 def api_runSimulation():
     r = _request_content(PKDict(fixup_old_data=True))
+    # TODO(e-carlin): I'm not sure where the best place to put this data on the request is
```

Comment:
@robnagler any advice on this? I'm not sure where the best place to put the rsmanifest is on the reqeust (if rsmanifest is on the request it will be used to set the image for the simulation)